com.github.alexr4535.siglo.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id" : "com.github.alexr4535.siglo",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "siglo",
+    "finish-args" : [
+        "--allow=bluetooth",
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=system-bus",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "gatt",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "pip3 install --prefix=/app --no-cache-dir gatt"
+            ]
+        },
+        {
+            "name": "dbus-python",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "pip3 install --prefix=/app --no-cache-dir dbus-python"
+            ]
+        },
+        {
+            "name": "python3-requests",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "pip3 install --prefix=/app --no-cache-dir requests"
+            ]
+        },
+        {
+            "name": "python3-pyxdg",
+            "buildsystem": "simple",
+            "build-options": {
+                "build-args": [
+                    "--share=network"
+                ]
+            },
+            "build-commands": [
+                "pip3 install --prefix=/app --no-cache-dir pyxdg"
+            ]
+        },
+        {
+            "name" : "siglo",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/alexr4535/siglo",
+                    "tag" : "v0.8.4"
```

Comment:
Please **add** a commit here to protect against re-tagging.

Reply:
i dont understand what you mean by this.

Reply:
Typically, you just need to add the commit hash of the particular commit in addition to the tag as shown here:

```json
"tag" : "v0.8.4",
"commit" : "b54a4e719853dfcf94473f9adea9adc87d102d10"
```